lib/sequelize.js

```diff
@@ -1145,7 +1145,7 @@ module.exports = (function() {
             return transaction.commit().then(function () {
               resolve(result);
             });
-          }).catch(function (err) {
+          }).then(null, function (err) {
```

Comment:
Doubt that, commit().then(...) doesn't have a onrejected handler, error in commit could lead to rollback. Write them in the same node may change the behavior.